scripts/add_existing_baseyear.py

```diff
@@ -464,6 +465,25 @@ def get_efficiency(
     return efficiency
 
 
+def valid_grouping_years(
+    grouping_years: Sequence[int] | pd.Index,
+    baseyear: int,
+    max_lifetime: float | None = None,
+) -> pd.Index:
+    grouping_years = pd.Index(sorted(grouping_years)).astype(int)
+
+    valid = grouping_years <= baseyear
```

Comment:
I am always confused about the type of `baseyear`, the old code explicitly used `int(baseyear)`, guess not needed as snakemake converts the dtype automatically of the wildcard to int?

Reply:
The same is actually true for `grouping_years` so that we could also get rid of the `.astype(int)`